### ISSUE TITLE
Feature request: Supress messages for existing devices

### ISSUE THREAD

**Issue by lawa42** [Cropi/usbguard-notifier#63]
It would be great to a have a config option that allows to suppress messages about known devices. Right now when connecting the laptop to the dock I get usbguard messages for all the usb devices that get connected. They are all mine and are all allowed. So it would be nice to have an option to reduce the noise a bit.

**Comment by muelli** [comment]
Excuse my ignorance, but what "messages" are you referring to?

Ah, I guess you mean a popup asking you to make a choice. In that case: sorry for the noise.
yeah, the USBGuard DBus API doesn't make it easy to catch that case.

**Comment by lawa42** [comment]
Yes this popup, and for me there are no choices. If I plug an unknown usb device the pop up just informs me that the device has been blocked. How do I get the choices?

**Comment by Cropi** [comment]
Hi @lawa42,
Thanks for the request. If I understand it correctly, you would like to have a configuration file to specify when should a notification pop up. For example do not show notifications when a device is plugged in AND device is allowed for use. Do I understand it correctly?

**Comment by lawa42** [comment]
Dear @Cropi 
yes thats exactly what I had in mind. If I move from home to work and back and connect the laptop to the docking station usb-hubs and devices (audio, mouse, keyboard, webcam, ...) get connected and disconnected which creates a lot of noise in the UI. The idea is to only have the most important messages displayed.

**Comment by Cropi** [comment]
I provided some ideas in https://github.com/Cropi/usbguard-notifier/issues/73#issuecomment-1349031650 . Let me know what you think about that if you are still interested :)
